app/package.json

```diff
@@ -68,6 +68,7 @@
     },
     "files": [
       "electron/main.js",
+      "electron/preload.js",
```

Comment:
Add this change to the commit that introduces the file.

Reply:
done